nemo/lightning/pytorch/callbacks/garbage_collection.py

```diff
@@ -0,0 +1,70 @@
+import gc
+from typing import Any
+
+import pytorch_lightning as pl
+from nemo.utils import logging
+
+
+class GarbageCollectionCallback(pl.Callback):
+    """Callback for synchronized manual Garbage Collection. This is required for distributed training
+    as all processes need to synchronize to garbage collect at the same time, without which
+    one process might hog or straggle all the rest of the processes.
+
+    Migration from NeMo 1.0:
+        When mitrating from NeMo1,
```

Comment:
`mitrating` -> `training`?

Reply:
Migration/Conversion if a NeMo 1.0 user is trying to use this. 

-> "Migrating a Training from NeMo 1.0". Does this explain it better ?